lenskit-core/src/main/java/org/grouplens/lenskit/data/text/DelimitedColumnEventFormat.java

```diff
@@ -51,8 +51,9 @@
     private int headerLines = 0;
 
     @Inject
-    public DelimitedColumnEventFormat(@Nonnull EventTypeDefinition etd) {
+    public DelimitedColumnEventFormat(@Nonnull EventTypeDefinition etd, @ColumnSeparator String delim) {
```

Comment:
Could technically be a breaking change if anyone was building this on their own instead of through injection.